netlify/functions/leaderboard.ts

```diff
@@ -34,30 +36,25 @@ const getLeaderboardResults = async (
 
   const filteredContests = allContests
     .filter(
-      (contest) => withinTimeframe(contest, contestRange) || !contestRange
+      (contest: DBContest) =>
+        withinTimeframe(contest, contestRange) || !contestRange
```

Comment:
wouldn't this work too, and avoid calling the helper function at all when there's a missing range?

```suggestion
        !contestRange || withinTimeframe(contest, contestRange)
```

but maybe there's a subtlety to the original logic that I'm missing where you would want to try check it?